src/Mod/Path/InitGui.py

```diff
@@ -158,7 +159,13 @@ def Initialize(self):
 
         if PathPreferences.advancedOCLFeaturesEnabled():
             try:
-                import ocl
+                subprocess.call(["camsim", "-v"])
```

Comment:
This is checking if camsim exist but i can't see camsim beeing used elsewhere. Perhaps calling camotics --version instead?

Reply:
Great point.  Yes, it's from an earlier version where I was using camsim.  I'll change it.